apps/src/templates/sectionProgress/SectionProgressToggle.jsx

```diff
@@ -80,15 +81,15 @@ class SectionProgressToggle extends React.Component {
           value={ViewType.SUMMARY}
           style={styles.toggleButton}
         >
-          <FontAwesome icon="search-minus" />
+          <div>Lessons</div>
```

Comment:
Can we i18n this string and "Levels" below?

Reply:
Good catch - will update